packages/main/src/components/AnalyticalTable/columnHeader/index.tsx

```diff
@@ -83,32 +88,41 @@ export const ColumnHeader: FC<ColumnHeaderProps> = (props) => {
     );
   }, [classes, column.filterValue, column.isSorted, column.isGrouped, column.isSortedDesc, children]);
 
-  style.width = '100%';
-  style.fontWeight = 'normal';
-  style.cursor = 'pointer';
-  style.height = '100%';
+  const innerStyle = { ...style };
```

Comment:
we should wrap everything related to creating the innerStyles object into useMemo